pkg/hubble/observer/local_observer.go

```diff
@@ -577,11 +589,22 @@ func (r *eventsReader) Next(ctx context.Context) (*v1.Event, error) {
 	}
 }
 
+func validateRequest(req genericRequest) error {
+	if req.GetFirst() && req.GetFollow() {
```

Comment:
does first work with time ranges?

Reply:
Hmm. I guess if it works without, then it should work with. Ie: getting the first 5 flows, within the time range.

Reply:
Actually, after looking into this, it looks like we only consider since and number exclusively. If since is set, then we iterate backwards until we reach an event with the timestamp, otherwise we iterate until we reach the specified number of events. So I think for now, we can ignore time ranges when first is set.